build.sbt

```diff
@@ -5,10 +5,20 @@ ThisBuild / crossScalaVersions := Seq("2.12.17", scala213, scala3)
 
 ThisBuild / tlBaseVersion := "2.5"
 
-ThisBuild / githubWorkflowJavaVersions := Seq(JavaSpec.temurin("11"))
+val temurin11 = JavaSpec.temurin("11")
+ThisBuild / githubWorkflowJavaVersions := Seq(temurin11)
 
-ThisBuild / githubWorkflowBuildMatrixExclusions += // TODO
-  MatrixExclude(Map("scala" -> scala3, "project" -> "rootJVM"))
+ThisBuild / githubWorkflowBuildMatrixExclusions ++= Seq(
+  MatrixExclude(Map("scala" -> scala3, "project" -> "rootJVM")), // TODO
+  MatrixExclude(
+    Map("scala" -> scala3, "project" -> "rootNative", "os" -> "ubuntu-latest")
+  ) // run on macOS instead
+)
+
+ThisBuild / githubWorkflowBuildMatrixInclusions +=
+  MatrixInclude(Map("scala" -> scala3, "java" -> temurin11.render, "project" -> "rootNative"),
+                Map("os"    -> "macos-latest")
```

Comment:
Ufff, interesting, so native can only run on macos?

Reply:
The problem is that on GHA, the free ubuntu runner only has 7 GB of RAM, but free macOS runner gets 14 GB of RAM. 7 GB just isn't enough for LLVM compiler I guess ...